src/components/apps/AgaveAuthPromptDialog.js

```diff
@@ -0,0 +1,70 @@
+/**
+ *
+ * @author sriram
+ * A simple dialog to prompt users to login with Agave
+ */
+
+import React from "react";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+import { useTheme } from "@material-ui/core/styles";
+import { build, getMessage, formatMessage, withI18N } from "@cyverse-de/ui-lib";
+import intlData from "./messages";
+import ids from "./ids";
+import { injectIntl } from "react-intl";
+
+function AgaveAuthPromptDialog(props) {
+    const { baseId, location, open, handleClose, intl } = props;
+    const theme = useTheme();
+    const fullScreen = useMediaQuery(theme.breakpoints.down("sm"));
+    const redirectUser = () => {
+        window.location.replace(location);
+        handleClose();
+    };
+    return (
+        <Dialog
+            open={open}
+            onClose={handleClose}
+            id={build(baseId, ids.AGAVE_AUTH_PROMPT_DIALOG)}
+            fullScreen={fullScreen}
+            aria-labelledby={formatMessage(intl, "agaveRedirectTitle")}
```

Comment:
`aria-labelledby` should contain an element ID, I believe.  This looks to be a shared message for the dialog title.  So in your case, you just have to add an ID to the `DialogTitle` component and add that same ID here in the `aria-labelledby`.